tests/e2e/machine_pool_test.go

```diff
@@ -4,53 +4,73 @@ import (
 
 	// nolint
 
+	"net/http"
 	"os"
 
-	EXE "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/exec"
-
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"github.com/openshift-online/ocm-sdk-go/logging"
+	cms "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/cms"
+	conn "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/connection"
+	con "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
+	exe "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/exec"
 )
 
+var logger logging.Logger
+
 var _ = Describe("TF Test", func() {
 	Describe("Create MachinePool test", func() {
 		var cluster_id string
+		var mpService *exe.MachinePoolService
 		BeforeEach(func() {
 			cluster_id = os.Getenv("CLUSTER_ID")
+			mpService = exe.NewMachinePoolService(con.MachinePoolDir)
 		})
 		AfterEach(func() {
+			err := mpService.Destroy()
+			Expect(err).ToNot(HaveOccurred())
 			return
 		})
-		It("MachinePoolExampleNegative", func() {
-
-			mpParam := &EXE.MachinePoolArgs{
-				Token: token,
-				// OCMENV:      "staging",
-				Cluster:     cluster_id,
-				Replicas:    3,
-				MachineType: "invalid",
-				Name:        "testmp",
-			}
-
-			_, err := EXE.CreateMyTFMachinePool(mpParam, "-auto-approve", "-no-color")
-			Expect(err).To(HaveOccurred())
-			Expect(err.Error()).Should(ContainSubstring("is not supported for cloud provider 'aws'"))
-		})
-		It("MachinePoolExamplePositive", func() {
-			mpParam := &EXE.MachinePoolArgs{
-				Token: token,
-				// OCMENV:      "staging",
-				Cluster:     cluster_id,
-				Replicas:    3,
-				MachineType: "r5.xlarge",
-				Name:        "testmp3",
-			}
-
-			_, err := EXE.CreateMyTFMachinePool(mpParam, "-no-color")
-			Expect(err).ToNot(HaveOccurred())
+		// It("MachinePoolExampleNegative", func() {
 
-			err = EXE.DestroyMyTFMachinePool(mpParam, "-no-color")
-			Expect(err).ToNot(HaveOccurred())
+		// 	mpParam := &exe.MachinePoolArgs{
+		// 		Token: token,
+		// 		// OCMENV:      "staging",
+		// 		Cluster:     cluster_id,
+		// 		Replicas:    3,
+		// 		MachineType: "invalid",
+		// 		Name:        "testmp",
+		// 	}
+
+		// 	_, err := exe.CreateMyTFMachinePool(mpParam, "-auto-approve", "-no-color")
+		// 	Expect(err).To(HaveOccurred())
+		// 	Expect(err.Error()).Should(ContainSubstring("is not supported for cloud provider 'aws'"))
+		// })
+		Context("Author:amalykhi-High-OCP-64757 @OCP-64757 @amalykhi", func() {
```

Comment:
Remove your tag name please

Reply:
Shouldn't we mark the owner of the TC?
That's how they work on this in OCM. @xueli181114 what do you think should we define tc owner for public repo tests?

Reply:
@bardielle Can we keep it for this MR? If it's a rule that we cannot have name tagged, we will figure a way to have mappings in future. But before we implement that, we need the tag to identity who should be responsible for the case failure debugging.